cli/dts/lib.deno.ns.d.ts

```diff
@@ -2263,9 +2263,42 @@ declare namespace Deno {
     kill(signo: string): void;
   }
 
-  // TODO(kt3k): Remove this type when std updated the signal usage.
+  // TODO(kt3k): Remove this type later.
```

Comment:
Why?

Reply:
Because we don't use `Signal` in other part (`kill`, `addSignalListener`, etc) anymore.

Do you think we should keep using Signal type in those params? (Maybe I read Bert's comment in a wrong way)

Reply:
Yes, I think we should.